### PR TITLE
Filter out non-serializable objects from payload

### PR DESCRIPTION
**Pull request by albertodebortoli**
As per title.

### CODE REVIEW

File: JustLog/Classes/Logger.swift [hunk 1]
```diff
@@ -211,13 +211,14 @@ extension Logger {
             }
             retVal[userInfoConst] = options
         }
-
         
         if let error = error {
-            retVal[errorsConst] = error.disassociatedErrorChain().map { return errorDictionary(for: $0) }
+            let errorDictionaries = error.disassociatedErrorChain()
+                .map { errorDictionary(for: $0) }
+                .filter { $0.values.allSatisfy { JSONSerialization.isValidJSONObject($0) } }
```
Comment (antoniostrijdom) [inline JustLog/Classes/Logger.swift:218]: What is performance like on `isValidJSONObject`? We will probably be calling this a lot.

Comment (albertodebortoli) [reply JustLog/Classes/Logger.swift:218]: Gotta do what you gotta do.

Comment (antoniostrijdom) [reply JustLog/Classes/Logger.swift:218]: Fair enough 😄

**Review by antoniostrijdom — APPROVED** [review]

**Review by dchakarov — APPROVED** [review]
LGTM

**Pull request merged**